### ISSUE TITLE
Reports not working for some players

### ISSUE THREAD

**Issue by MatStef132** [BadgerCode/tttdamagelogs#18]
Hello!
When some players try to report people they get this error and nothing happens. 
I have no idea why it happens only to specific people. They tried reinstalling gmod completly and it did nothing.
```[tttdamagelogs] lua/includes/extensions/table.lua:102: bad argument #1 to 'pairs' (table expected, got nil)
  1. pairs - [C]:-1
   2. table_HasValue - lua/includes/extensions/table.lua:102
    3. HandlePlayerReport - addons/tttdamagelogs/lua/damagelogs/server/rdm_manager.lua:373
     4. func - addons/tttdamagelogs/lua/damagelogs/server/rdm_manager.lua:508
      5. unknown - lua/includes/extensions/net.lua:32```

**Comment by BadgerCode** [comment]
Hey.
Could you confirm which version of the damage logs you are using?
It should be visible at the top of the damage logs window

![Preview](https://i.imgur.com/9eFdulZ.png)

Affected lines:
https://github.com/BadgerCode/tttdamagelogs/blob/4a0ca3f9fed53603556760ab59ef45f7be4a135a/lua/damagelogs/server/rdm_manager.lua#L373-L376

**Comment by MatStef132** [comment]
I was using both the one from the releases and the master version, both of them didn't work.

**Comment by BadgerCode** [comment]
Thank you! I'll take a look into this and see if I can spot anything obvious.

Does the error happen frequently?
Did this happen immediately after installing the addon?
Did you restart the server/change the map after installing the addon?

**Comment by MatStef132** [comment]
1. This error happens always when some players (currently 2) try to report
2. It wasn't happening for them until a mapchange and then always happened
3. I both restarted the server and changed the map

**Comment by BadgerCode** [comment]
I've made a patch that should fix this problem.
#20 

I can't say when I will be able to release it.
I have also noticed another issue that I wish to fix.

You can download it here:
https://github.com/BadgerCode/tttdamagelogs/archive/Iss-18_report_limit.zip

**Comment by BadgerCode** [comment]
The fix for this (#20) should be shipped in the next release of this addon.

Feel free to comment on this issue if you still experience the issue once the next version has been released and you are using it.
